### PR TITLE
[132] Strip out front end tooling

### PR DESCRIPTION
**Pull request by tm-kn**
https://projects.torchbox.com/projects/tbxcom-redesign/tickets/132

- Also set the home view to be 404.
- Wagtail views only accessible to the logged-in users.
- robots.txt to ignore the wagtail views as well.

### CODE REVIEW

File: tbx/urls.py [hunk 1]
```diff
@@ -52,6 +54,7 @@
     path('review', include(wagtailreview_urls)),
```
Comment (zerolab) [inline tbx/urls.py]: Do reviews still work with the headless/FE setup?

File: tbx/urls.py [hunk 2]
```diff
@@ -71,8 +74,7 @@
...
 
 # Join private and public URLs.
-urlpatterns = private_urlpatterns + urlpatterns + [
-    # Add Wagtail URLs at the end.
-    # Wagtail cache-control is set on the page models's serve methods.
-    path(r'', include(wagtail_urls)),
-]
+urlpatterns = private_urlpatterns + urlpatterns + decorate_urlpatterns([
+    # Make wagtail paths required by login.
+    path('wagtail/', include(wagtail_urls))
```
Comment (zerolab) [inline tbx/urls.py]: Shouldn't this change be documented/highlighted in the README?

File: Dockerfile [hunk 3]
```diff
@@ -14,34 +14,15 @@ EXPOSE 8000
...
 RUN pip install -r requirements.txt
 
-# Copy front-end code
-WORKDIR /app/tbx/core/static_src
-COPY tbx/core/static_src .
-
-# Compile static files
-RUN npm run build:prod
-
-WORKDIR /app
```
Comment (zerolab) [inline Dockerfile]: What about node in the VM? https://github.com/torchbox/wagtail-torchbox/blob/405e1a02480ab21b7da65ce26c8e20bdb4a3fcd4/vagrant/provision.sh#L67

File: tbx/urls.py (hunk 1 above)
Comment (zerolab) [reply tbx/urls.py]: btw, this is more of an observation, rather than something to do

File: tbx/urls.py (hunk 2 above)
Comment (tm-kn) [reply tbx/urls.py]: Updated. Better comment used as well.

File: tbx/urls.py (hunk 1 above)
Comment (tm-kn) [reply tbx/urls.py]: It should since it uses /admin/ URL. I'll test on staging once you clear the review.

File: Dockerfile (hunk 3 above)
Comment (tm-kn) [reply Dockerfile]: Cheers, sorted.

**Comment by zerolab** [comment]
Do we need to leave the templates at all?

**Comment by tm-kn** [comment]
> Do we need to leave the templates at all?

Let's leave them for now. Otherwise Wagtail "live" links will lead to HTTP 500 and not sure how that affects preview too. May be a good one for later.

File: tbx/urls.py (hunk 1 above)
Comment (tm-kn) [reply tbx/urls.py]: Previews work! :tada:

**Pull request merged**